ci-operator/config/maistra/istio-images-centos/maistra-istio-images-centos-maistra-2.2.yaml

```diff
@@ -12,7 +12,7 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: istio-images-centos-lint
+- as: istio-images-centos-lint-maistra-2-2
```

Comment:
Why not having just the version, without the word `maistra`? (This applies to everything below).

Also, are you sure there's no way to have dots in the name?

Reply:
yes, I will remove the word there. 
I am sure about the dots problem. I saw a message before. It's something about their tag name
and yaml syntax issues from their converter using dots.

error="failed to ensure imagestreamtags in cluster build01: failed waiting for imagestreamtag build-cache/maistra-api:maistra-2.3 to appear: timed out waiting for the condition" org=openshift repo=release